components/registry-facade/cmd/run.go

```diff
@@ -134,3 +138,75 @@ func authorizerFromDockerConfig(cfg *configfile.ConfigFile) docker.Authorizer {
 		return
 	}))
 }
+
+// watchConfig watches the configuration file and if changed reloads the static layer
+func watchConfig(fn string, reg *registry.Registry) {
```

Comment:
I have a preference towards longer variable names, so something lie `configPath` (if it is a path) instead of `fn`

Reply:
Usually I'd agree, although in go `fn` is a fairly standard name choice and in combination with the `watchConfig` function its semantic is pretty clear I'd think